ml-model/model/image_to_latex.py

```diff
@@ -0,0 +1,22 @@
+from rapid_latex_ocr import LatexOCR
+
```

Comment:
Can you delete this files from the PR? Lots of people have made changes on these files and will screw it up